### ISSUE TITLE
PDOException: SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'dateCreated' in order clause is ambiguous

### ISSUE THREAD

**Issue by Rias500** [solspace/craft3-freeform#3]
This error happens when accessing the submissions, Craft shows an "unknown error has occured"

*Stacktrace*
```
2018-01-24 21:29:27 [127.0.0.1][1][-][error][yii\db\IntegrityException] PDOException: SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'dateCreated' in order clause is ambiguous in /Users/rias/Code/rias/vendor/yiisoft/yii2/db/Command.php:1067
Stack trace:
#0 /Users/rias/Code/rias/vendor/yiisoft/yii2/db/Command.php(1067): PDOStatement->execute()
#1 /Users/rias/Code/rias/vendor/yiisoft/yii2/db/Command.php(386): yii\db\Command->queryInternal('fetchAll', NULL)
#2 /Users/rias/Code/rias/vendor/yiisoft/yii2/db/Query.php(217): yii\db\Command->queryAll()
#3 /Users/rias/Code/rias/vendor/craftcms/cms/src/db/Query.php(118): yii\db\Query->all(NULL)
#4 /Users/rias/Code/rias/vendor/craftcms/cms/src/elements/db/ElementQuery.php(1003): craft\db\Query->all(NULL)
#5 /Users/rias/Code/rias/vendor/craftcms/cms/src/base/Element.php(442): craft\elements\db\ElementQuery->all()
#6 /Users/rias/Code/rias/vendor/craftcms/cms/src/controllers/ElementIndexesController.php(391): craft\base\Element::indexHtml(Object(Solspace\Freeform\Elements\Db\SubmissionQuery), Array, Array, '*', 'index', true, true)
#7 /Users/rias/Code/rias/vendor/craftcms/cms/src/controllers/ElementIndexesController.php(119): craft\controllers\ElementIndexesController->_elementResponseData(true, true)
#8 [internal function]: craft\controllers\ElementIndexesController->actionGetElements()
#9 /Users/rias/Code/rias/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#10 /Users/rias/Code/rias/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#11 /Users/rias/Code/rias/vendor/craftcms/cms/src/web/Controller.php(80): yii\base\Controller->runAction('get-elements', Array)
#12 /Users/rias/Code/rias/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('get-elements', Array)
#13 /Users/rias/Code/rias/vendor/craftcms/cms/src/web/Application.php(251): yii\base\Module->runAction('element-indexes...', Array)
#14 /Users/rias/Code/rias/vendor/craftcms/cms/src/web/Application.php(453): craft\web\Application->runAction('element-indexes...', Array)
#15 /Users/rias/Code/rias/vendor/craftcms/cms/src/web/Application.php(211): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#16 /Users/rias/Code/rias/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#17 /Users/rias/Code/rias/web/index.php(21): yii\base\Application->run()
#18 /Users/rias/.composer/vendor/laravel/valet/server.php(133): require('/Users/rias/Cod...')
#19 {main}
```

**Comment by kjmartens** [comment]
Thanks for reporting. I can duplicate this one as well. For some reason it wasn't an issue before we launched. Sorry for the trouble.

**Comment by kjmartens** [comment]
This should be resolved now in Freeform 2.0.0-beta.2 :)
Please updates and let me know if the issue persists.
